doc-changelog/action.yml

```diff
@@ -0,0 +1,138 @@
+name: >
+  Documentation Changelog
+
+description: |
+  Adds a newsfragment to the target repository using ``towncrier``
+  containing the title and number of the pull request.
```

Comment:
It will be nice to add an example here or a link to the documentation where you can see an actual implementation of it.

Reply:
You can create a new example for this action by creating a file named `doc-changelog-basic.yml` inside the `doc/source/doc-actions/examples/` directory, @klmcadams.